packages/vscode/src/extension.ts

```diff
@@ -98,18 +103,10 @@ export async function activate(context: ExtensionContext): Promise<void> {
     },
   }
 
-  // enable fileWatcher to watch .prisma folder inside node_modules
-  if (!isTest) {
-    await enablePrismaNodeModulesFolderWatch()
-  }
-
   // Options to control the language client
   const clientOptions: LanguageClientOptions = {
     // Register the server for prisma documents
     documentSelector: [{ scheme: 'file', language: 'prisma' }],
-    synchronize: {
-      fileEvents: workspace.createFileSystemWatcher('**/.prisma/client/*.d.ts'),
```

Comment:
@alexdima 
This is the section where we used to use VS Code's built-in file watcher.